src/test/unit/comparison.test.js

```diff
@@ -30,4 +30,42 @@ describe('mergeProfiles function', function() {
 
     expect(mergedProfile.profile.meta.interval).toEqual(10);
   });
+
+  it('should set the resulting profile to symbolicated if all are symbolicated', () => {
+    const { profile: symbolicatedProfile } = getProfileFromTextSamples(`A`);
```

Comment:
Just curious why you're mixing and matching `'` and ` characters for strings I noticed this in a few places.

Reply:
Here I copied this from the existing test above without thinking too much about it.
The existing test above has been written by a contributor and I didn't feel like asking him to change the string delimiter :)

Usually except for templating, I use \` when there's a `'` in the string (especially in test descriptions). Sometimes I use `"` too. I mean I don't really mind :-)

Reply:
I changed all \` to `'` in this file.